docs/API_walkthrough.md

```diff
@@ -0,0 +1,620 @@
+```
+
+* Adding the options `KeyValues` to the parameter of a get query will return a reduced version of the Entity providing only top level attribute and their value or object.
+
+```shell
+http http://localhost:8080/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01  options==keyValues Link:$CONTEXT_LINK
+```
+Sample payload returned showing a reduced version of the entity BeeHive:
+
+```json
+{
+    "id": "urn:ngsi-ld:BeeHive:01",
+    "type": "BeeHive",
+    "humidity": 60,
+    "temperature": 22.2,
+    "belongs": "urn:ngsi-ld:Apiary:01",
+    "managedBy": "urn:ngsi-ld:Beekeeper:01",
+    "location": {
+        "type": "Point",
+        "coordinates": [
+            24.30623,
+            60.07966
+        ]
+    },
+    "@context": [
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+    ]
+}
+```
+
+* Let's add a name to the created beehive:
+
+```shell
+http POST http://localhost:8080/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01/attrs \
+    Link:$CONTEXT_LINK < beehive_addName.json
+```
+
+beehive_addName.json
+```json
+{
+   "name":{
+      "type":"Property",
+      "value":"BeeHiveSophia"
+   }
+}
+```
+
+* The recently added name property can be deleted:
+
+```shell
+http DELETE http://localhost:8080/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01/attrs/name Link:$CONTEXT_LINK
+```
+
+* Let's create a subscription on the Beehive entity. We would like to be notified when the temperature of the BeeHive exceeds 40. To do so, we need a working `endpoint` in order to receive the notification related to the subscription. For this example, we are using Post Server V2 [http://ptsv2.com/](http://ptsv2.com/). This is a free public service. It is used only for tests and debugs. You need to configure your appropriate working `endpoint` for your private data.
+
+```shell
+http POST http://localhost:8080/ngsi-ld/v1/subscriptions Content-Type:application/ld+json < subscription_to_beehive.jsonld
+```
+
+subscription_to_beehive.jsonld
+```json
+{
+  "id":"urn:ngsi-ld:Subscription:01",
+  "type":"Subscription",
+  "entities": [
+    {
+      "type": "BeeHive"
+    }
+  ],
+  "q": "temperature>40",
+  "notification": {
+    "attributes": ["temperature"],
+    "format": "normalized",
+    "endpoint": {
+      "uri": "http://ptsv2.com/t/ir6uu-1626449561/post",
+      "accept": "application/json"
+    }
+  },
+  "@context": [
+     "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+  ]
+}
+```
+
+* The created subscription can be retrieved by id:
+
+```shell
+http http://localhost:8080/ngsi-ld/v1/subscriptions/urn:ngsi-ld:Subscription:01 Link:$CONTEXT_LINK
+```
+
+* We can update the temperature with a value grater then 40, using the following query:
+
+```shell
+http PATCH http://localhost:8080/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01/attrs \
```

Comment:
maybe a partial attribute update?

Reply:
no answer?

Reply:
Didn't got it (I thought that it was about the title of the section)